data/human/human missions.txt

```diff
@@ -2502,6 +2502,123 @@ event "terraforming follow-up"
 
 
 
+mission "Terraforming 13"
+	minor
+	name "Field Trip"
+	description "Amy, who you helped terraform Rand and Tundra, seems to share some of her terraforming philosophy with the Wanderers. Bring her to <destination> to visit them."
+	source "Tundra"
+	destination "Desi Seledrak"
+	passengers 1
+	to offer
+		has "event: terraforming follow-up"
+		has "wanderers sestor done"
+	on offer
+		conversation
+			`As you walk through the spaceport, you notice a poster featuring Amy, whom you previously helped terraform this world and Rand. The poster describes that terraforming action, and the long-term goal to restore the warmer climate Tundra had millions of years ago. Amy's work on "equilibrium mapping," the idea that planets naturally "want" to support more complex and varied life, seems similar to the Wanderer philosophy that each planet has its own "will", and that the best way to terraform it is to work with that "will." Amy may be interested in learning about the Wanderers and their techniques. Would you like to find her and tell her about them?`
+			choice
+				`	(Yes.)`
+				`	(No.)`
+					decline
+			`	You're able to contact Amy. She has been continuing to monitor the status of Tundra's climate, alongside the researchers from the Deep who she worked with to terraform this world. You tell her you've encountered some alien terraformers who may share her philosophy. She is eager to learn more about them and agrees to meet you in the spaceport bar.`
+			`	Once she arrives, you explain about the Wanderers: that the work of their civilisation is to repair worlds left in ruin by their previous inhabitants. And their view that those worlds have their own identities that they must collaborate with them in order for terraforming to be successful.`
+			`	She has a slew of questions ranging from how the Wanderers determine a planet's identity to how often asteroids are used in their work. Unfortunately, you are not able to offer much information about the specific details of how the Wanderers go about their terraforming work. Disappointed, but not defeated, Amy asks, "Could you take me to visit these aliens?`
```

Comment:
```suggestion
			`	She has a slew of questions ranging from how the Wanderers determine a planet's identity to how often asteroids are used in their work. Unfortunately, you are not able to offer much information about the specific details of how the Wanderers go about their terraforming work. Disappointed, but not defeated, Amy asks, "Could you take me to visit these aliens?”`
```

Reply:
Your suggestion contains a "fancy" speech mark; we don't use those.
```suggestion
			`	She has a slew of questions ranging from how the Wanderers determine a planet's identity to how often asteroids are used in their work. Unfortunately, you are not able to offer much information about the specific details of how the Wanderers go about their terraforming work. Disappointed, but not defeated, Amy asks, "Could you take me to visit these aliens?"`
```